dnf5/bash-completion/dnf5

```diff
@@ -7,4 +7,11 @@ _do_dnf5_completion()
     mapfile -t COMPREPLY <<<$("${1}" "--complete=${cword}" "${words[@]}")
 }
 
-complete -F _do_dnf5_completion -o nosort -o nospace dnf5
+complete_cmds="dnf5"
+dnf_target=$(readlink -f "/usr/bin/dnf")
```

Comment:
Add "Requires: coreutils" into dnf5.spec for the readlink tool.

Reply:
Thanks, I did the same change for the dnf pull request.